src/main/java/io/confluent/connect/hdfs/avro/AvroRecordWriterProvider.java

```diff
@@ -71,7 +73,7 @@ public void write(SinkRecord record) {
           }
         }
 
-        log.trace("Sink record: {}", record.toString());
+        log.trace("Sink record: {}", record);
```

Comment:
removed the .toString() method call because it is redundant and log level will be higher than Trace in most situations (we can save some CPU cycles).

Reply:
Makes sense, `sfl4j` implementation will call this anyway.